core/events/workspace_events.ts

```diff
@@ -54,17 +54,17 @@ export class FinishedLoading extends AbstractEvent {
     }
     json['workspaceId'] = this.workspaceId;
     return json;
-  }
+  } */
 
   /**
    * Decode the JSON event.
    *
    * @param json JSON representation.
    */
-  override fromJson(json: FinishedLoadingJson) {
+  /* override fromJson(json: FinishedLoadingJson) {
     super.fromJson(json);
     this.workspaceId = json['workspaceId'];
-  }
+  } */
```

Comment:
(This method and the one above, since that's not clear from the little snippet attached to this comment).

Reply:
Yes, Beka pointed out that serializing this event makes no sense, since a workspace finishing loading is not relevant/replayable on another one. Thanks for flagging!